src/utils/consts.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import cappv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+
+var cappAPIGroup = cappv1alpha1.GroupVersion.Group
+
+const (
+	TokenRequestSuffix = "token-request"
+	SecretKind         = "Secret"
+	ServiceAccountKind = "ServiceAccount"
+	TokenAudienceAPI   = "api"
+	TokenExpiration    = 36000
+)
```

Comment:
Some of these constants are only being used in a single file so they should just be defined there instead of in its own file.